### ISSUE TITLE
[Bug]: guile3 crashes with a "Pre-boot error" on armv8l Android (32 bits?)

### ISSUE THREAD

**Issue by mtarenskeen** [termux/termux-packages#14625]
### Problem description

On my Samsung Galaxy A tablet (2016) I can't run lilypond because guile3 always crashes with the following error message:

`Pre-boot error; key: out-of-range, args: (#f "Value out of range ~S to< ~S" (0 -1 1) (1)Aborted`

However, on my Huawei P10 Lite guile3 and lilypond run fine. But this is on aarch64 Android, whereas my older Samsung tablet is armv8l Android. My guess is that's the problem. Can anyone reproduce (and hopefully fix) this?

### What steps will reproduce the bug?

In the termux terminal type

`guile -v`

or

`guile -h`

or any other guile command.

### What is the expected behavior?

guile showing version number (-v) or a helptext (-h) or doing something useful (like when running lilypond on a lilypond .ly file)

### System information

termux-info:

```
Termux Variables:
TERMUX_API_VERSION=0.50.1
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=18192
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
arm
Subscribed repositories:
# sources.list
deb https://mirror.accum.se/mirror/termux.dev/termux-main stable main
Updatable packages:
diffutils/stable 3.9 arm [upgradable from: 3.8]
python-pip/stable 22.3.1-3 all [upgradable from: 22.3.1-1]
units/stable 2.22-1 arm [upgradable from: 2.22]
termux-tools version:
1.35.0
Android version:
8.1.0
Kernel build information:
Linux localhost 3.18.14-15260192 #1 SMP PREEMPT Tue Nov 3 16:08:25 KST 2020 armv8l Android
Device manufacturer:
samsung
Device model:
SM-T580
```

**Comment by Grimler91** [comment]
I can reproduce it on my arm device (but not aarch64), so probably 32bit or arm specific issue

**Comment by mtarenskeen** [comment]
Thanks for the fix! Guile and Lilypond are working perfectly now even on my 32bit tablet.-------- Oorspronkelijk bericht --------Onderwerp: Re: [termux/termux-packages] [Bug]: guile3 crashes with a "Pre-boot error" on armv8l Android (32 bits?) (Issue #14625)Van: xtkoba Aan: termux/termux-packages Cc: Martin Tarenskeen ,Author 
Closed #14625 as completed via #14659.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/termux/termux-packages/issues/14625#event-8310202806",
"url": "https://github.com/termux/termux-packages/issues/14625#event-8310202806",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
